src/server/slowlog.h

```diff
@@ -0,0 +1,70 @@
+// Copyright 2022, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+
+#pragma once
+
+#include <boost/circular_buffer.hpp>
+#include <string>
+#include <vector>
+
+#include "base/integral_types.h"
+#include "server/common.h"
+namespace dfly {
+
+class SlowlogEntry {
+ public:
+  SlowlogEntry(uint64_t command_id, uint64_t unix_timestamp, uint64_t execution_time_micro,
+               std::vector<std::string> cmd_args, std::string client_ip, std::string client_name);
+
+  const std::vector<std::string>& cmdArgs() const {
+    return cmd_args_;
+  }
+  int64 commandId() const {
+    return command_id_;
+  }
+  int64 unixTimestamp() const {
```

Comment:
thank you for submitting this PR! I have not went over all the code but I have few comments that worth fixing across the whole change before I do deep dive.
1. naming conventions: `FunctionName`, `this_is_var_`. Please check Google styleguide for more info.  
2. int64 -> int64_t

Reply:
Went over the style guide, it's quite extensive, so not sure if I managed to internalize everything. But I fixed the following issues:
1. usage of `int64_t` instead of `int64`
2. switched unsigned integer to signed ones
3. consistently use FunctionNames camel case and member variables as `snake-case_`